x-pack/legacy/plugins/integrations_manager/server/index.ts

```diff
@@ -3,7 +3,12 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import { Plugin, PluginInitializerContext } from './plugin';
+import { PluginInitializerContext } from 'src/core/server';
```

Comment:
```suggestion
import { Plugin, PluginInitializerContext } from './plugin';
```

I'd like to have the Plugin in control of its types. This way it can make any changes to import location, shape, etc and other files won't be affected. Or, rather, as minimally as possible.